frontend/src/views/AdminEventLog.vue

```diff
@@ -1,195 +0,0 @@
-import UsersRepository from "../api/UsersRepository";
-import ClientsRepository from "../api/ClientsRepository";
-
-const options = {dateStyle: 'short', timeStyle: 'short'};
-    const formatDate = new Intl.DateTimeFormat(undefined, options).format;
-
-    export default {
-        name: "AdminEventLog",
-        data() {
-            return {
-                searchUserId: '',
-                searchClientId: '',
-                searchAdministratorId: '',
-                searchDateFrom: '',
-                searchDateTo: '',
-                filterEvents: '',
-                loadingStatus: false,
-                singleExpand: true,
-                adminEvents: [],
-                headers: [
-                    { text: 'Time', value: 'readableDate'},
-                    { text: 'Administrator', value: 'authDetails.fullName'},
-                    { text: 'Event type', value: 'operationType' },
-                    { text: 'Resource type', value: 'resourceType' },
-                    { text: 'Username', value: 'username' },
-                    { text: 'Application', value: 'clientName' },
-                    { text: 'Details', value: 'data-table-expand' },
-                ],
-                pageValue: 1,
-                expandedValues: [],
-            }
-        },
-
-        created() {
-            this.getAllEvents();
-        },
-
-        methods: {
-          searchEvents: function () {
-            const params = buildQueryParameters.call(this);
-            this.getEvents(() => AdminEventsRepository.getEvents(params));
-          },
-          getAllEvents: function () {
-            this.getEvents(AdminEventsRepository.getEvents);
-          },
-          getEvents: async function(getEvents) {
-            this.loadingStatus = true;
-            try {
-              let promise = await getEvents();
-              this.adminEvents = promise.data;
-              for (let [index, e] of this.adminEvents.entries()) {
-                e.key = index;
-                e.readableDate = formatDate(e.time);
-                // Assumes format users/nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn
-                // If we ever display non-users path events, we'll need to change this.
-                e.userId = e.resourcePath.substring(6, 42);
-                // CREATE and UPDATE USER events aren't associated with an application, so check for the "clients" path before extracting it.
-                if (e.resourcePath.includes('role-mappings/clients')) {
-                  // Assumes format users/.../role-mappings/clients/nnnnnnnn-nnnn-nnnn-nnnn-nnnnnnnnnnnn/
-                  e.clientId = e.resourcePath.substring(65, 101);
-                }
-              }
-              await ClientsRepository.addClientNamesToEvents(this.adminEvents);
-              await UsersRepository.addUsernamesToEvents(this.adminEvents);
-              await UsersRepository.addAdminNamesToEvents(this.adminEvents);
```

Comment:
I think you can also remove `ClientsRepository.addClientNamesToEvents`, `UsersRepository.addUsernamesToEvents`, and `addAdminNamesToEvents`, but please confirm they are not used anywhere else.